### PR TITLE
bottle: ignore go libexec path for stuff using go.

### PR DESCRIPTION
**Pull request by mikemcquaid**
These paths cannot be removed and are just used for debug stacktraces so seem to not be harmful for us to selectively ignore.

CC @Homebrew/owners for review/objections.

### CODE REVIEW

File: Library/Homebrew/cmd/bottle.rb [hunk 1]
```diff
@@ -163,8 +160,14 @@ def bottle_formula f
           prefix_check = prefix
         end
 
-        relocatable = !keg_contains(prefix_check, keg)
-        relocatable = !keg_contains(cellar, keg) && relocatable
+        options = {}
+        if f.deps.map(&:to_s).include? "go"
```
Comment (jacknagel) [inline Library/Homebrew/cmd/bottle.rb]: Maybe `f.deps.any? { |d| d.name == "go" }`

File: Library/Homebrew/cmd/bottle.rb [hunk 2]
```diff
@@ -33,25 +33,19 @@
...
-
+  def keg_contains string, keg, options={}
     result = false
     index = 0
 
     keg.each_unique_file_matching(string) do |file|
-      if ARGV.verbose?
-        opoo "String '#{string}' still exists in these files:" if index.zero?
-        puts "#{Tty.red}#{file}#{Tty.reset}"
-      end
```
Comment (jacknagel) [inline Library/Homebrew/cmd/bottle.rb]: This was somewhat useful, is there no way to preserve it?

**Comment by jacknagel** [comment]
Can we just pass ignores as parameter instead of using an options hash?

**Comment by mikemcquaid** [comment]
> Can we just pass ignores as parameter instead of using an options hash?

Sure.

File: Library/Homebrew/cmd/bottle.rb (hunk 2 above)
Comment (mikemcquaid) [inline Library/Homebrew/cmd/bottle.rb]: So, the problem I found with this was that it was effectively duplicated by the output from below. I'm open to preserving it but I couldn't see a way of doing so.

File: Library/Homebrew/cmd/bottle.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Homebrew/cmd/bottle.rb]: Cool.

File: Library/Homebrew/cmd/bottle.rb (hunk 2 above)
Comment (jacknagel) [inline Library/Homebrew/cmd/bottle.rb]: At least the filename would be good to have, it doesn't look like that's printed below (ignore me if it is)

Comment (mikemcquaid) [inline Library/Homebrew/cmd/bottle.rb]: Will double check and make sure we include it.

**Comment by mikemcquaid** [comment]
Addressed all comments and will merge this once green.

**Pull request closed without merging**